README.md

```diff
@@ -13,9 +13,70 @@ breeze.
 
 _Learn more at [bazel-ios.github.io](https://bazel-ios.github.io/)_
 
-### iOS Applications
+## Reference documentation
+
+[Click here](https://github.com/bazel-ios/rules_ios/tree/master/docs)
+for the documentation.
 
-_it supports all the primitives like apps, extensions, and widgets_
+## Getting started
+
+### WORKSPACE setup
+
+Add the following lines to your `WORKSPACE` file.
+
+```python
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# See the Releases page on GitHub for latest versions
```

Comment:
Could use https://github.com/bazel-ios/rules_ios/releases/latest once the Github Release is actually published.

Reply:
I'll add the link now good call